### PR TITLE
Adding a link to more info on Android Drawables

### PR DESCRIPTION
**Pull request by sfshaza2**
Fixes https://github.com/flutter/website/issues/9038

@domesticmouse, do you know who is best to review this technically? Maybe you?  :D

### CODE REVIEW

**Comment by sfshaza2** [comment]
@johnmccutchan, I know this isn't strictly your bailiwick, but (as a Flutter Android user) does this change seem ok?

File: src/platform-integration/android/splash-screen.md [hunk 1]
```diff
@@ -82,6 +88,8 @@ background color of the Flutter UI.
 </style>
 ```
 
+[drawables]: {{site.android-dev}}/studio/write/resource-manager#import
```
Comment (domesticmouse) [inline src/platform-integration/android/splash-screen.md:91]: That link seems to talk about how to import Drawables into an Android project using Android Studio.

Maybe https://developer.android.com/guide/topics/resources/drawable-resource is a better URL to send developers to?

Comment (sfshaza2) [reply src/platform-integration/android/splash-screen.md:91]: I saw that page, too. I thought maybe adding it to the project was needed? I mean, a Drawable itself is pretty normal.

Comment (domesticmouse) [reply src/platform-integration/android/splash-screen.md:91]: Ahh, Okiday

**Comment by johnmccutchan** [comment]
Adding @camsim99 who recently touched splash screens.

File: src/platform-integration/android/splash-screen.md [hunk 2]
```diff
@@ -55,6 +55,12 @@ operating system sets up the app's process.
 Android provides the concept of a [launch screen][] to
 display a `Drawable` while the app is initializing.
 
+A `Drawable` is an Android graphic.
+For information on adding a `Drawable` to your
```
Comment (atsansone) [inline src/platform-integration/android/splash-screen.md:59]: **nit:** Make this an action.
```suggestion
To learn how to add a `Drawable` to your
```

**Review by atsansone — APPROVED** [review]
LGTM % 1 nit.

**Comment by sfshaza2** [comment]
I'm landing once the build is happy. @camsim99, if you have any further info or thoughts, please let me know!

**Pull request merged**